packages/icons/20/options-icon.tsx

```diff
@@ -7,16 +7,10 @@ const SvgOptionsIcon = (props: SvgProps) => {
   const { color: colorToken = 'currentColor', ...rest } = props
   const color = useCurrentColor(colorToken)
   return (
-    <Svg
-      width={20}
-      height={20}
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
```

Comment:
I know `react-native-svg` is erroring on `xmlns`, but AFAIK it is necessary for browsers using SVG as a file.

It is probably better to patch `react-native-svg` types like [here](https://github.com/software-mansion/react-native-svg/issues/1638#issuecomment-924957643).